poetry.lock

```diff
@@ -303,11 +284,11 @@ numpy = ">=1.18,<1.23"
 
 [[package]]
 name = "numpy"
-version = "1.21.6"
+version = "1.22.4"
 description = "NumPy is the fundamental package for array computing with Python."
 category = "main"
 optional = false
-python-versions = ">=3.7,<3.11"
+python-versions = ">=3.8"
```

Comment:
It's probably worth keeping in the maximum version as well, in case we run into more issues with changes breaking our code or (more likely) one of our dependencies.

Reply:
Agreed. Python 3.11 is not released yet, but as soon as it is we can pretty easily confirm it works on CI and then support it.